config/index.ts

```diff
@@ -0,0 +1,36 @@
+// import { EEnvironments } from '@/utils/enums/general.enum';
+// import path from 'path';
+// import ConfigApp from '@/utils/helpers/Config';
+import config, { IConfig } from '@/config/environments/all';
+import { merge as _merge } from 'lodash';
+
+// const env = process.env.ENVIRONMENT;
```

Comment:
I created an issue to clean up and refactor the config implementation: https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/issues/12

Reply:
I removed the configs on the other PR. Just leaving one single file.